include/sir/platform.h

```diff
@@ -235,7 +249,8 @@ _set_thread_local_invalid_parameter_handler(
 #  endif
 # endif
 
-# if !defined(__MACOS__) && !defined(__BSD__) && !defined(__SOLARIS__)
+# if !defined(__MACOS__) && !defined(__BSD__) && !defined(__SOLARIS__) && \
+     !(defined(__GLIBC__) && GLIBC_VERSION >= 23800)
```

Comment:
It's correct it's, not backwards, but I forgot that I added a definition to `0` so it's not needed to check - it can be clearer.

There are non-glibc libc's that do define __GLIBC__ which is why I had to do that, let me take advantage of that.

Reply:
It can be 0, so need to check for glibc too, it's correc.t